3.11.22/js/pull.js

```diff
@@ -0,0 +1,43 @@
+let userBorn = prompt('What year of your born?')
+let age = 2022 - userBorn
+if(age === 2022 - null) {
+    alert('It is a pity that we did not get an answer')
+}
+let userCity = prompt('Which city do you live?')
+if(userCity === null) {
+    alert('It is a pity that we did not get an answer')
+}
+let userSport = prompt('What is your favorite sport?')
+if(userSport === null) {
+    alert('It is a pity that we did not get an answer')
+}
+// if !userBorn {} -> 0, null, ""
+
+if (age === 2022) {
+    age = ('not specified')}
+    else{
+  age = (`${'Your age is'} ${age} ${'years'}`)}
+
+if (userCity === 'Kyiv') {
```

Comment:
don't forget about typos, it will be better to check all data in lowercase, to avoid the problem with 
userCity = 'kyiv'
userCity = 'KYiv'
...